src/App.js

```diff
@@ -3,89 +3,20 @@ import 'bootstrap/dist/css/bootstrap.min.css'
 import NavBar from './components/Navbar'
 import Resources from './components/Resources'
 import Visualization from './components/Visualization'
-import {BrowserRouter as Router, Switch, Route} from 'react-router-dom' 
```

Comment:
It's best practice in javascript to add spaces between the elements and braces in object destructuring. For example:
```javascript
const { someProperty } = somObject;
```